File: Tests/Python/LB_plotly.py
import plotly.graph_objects as go
import numpy as np

# Generating data points
num_points = 20
y1 = np.random.randint(5, 15, num_points)  # Oxygen levels
y2 = np.random.randint(70, 120, num_points)  # Blood pressure

# Create traces for oxygen levels and blood pressure
trace1 = go.Scatter(y=y1, mode='lines+markers', name='Zuurstof', line=dict(color='blue'))
trace2 = go.Scatter(y=y2, mode='lines+markers', name='Bloeddruk', line=dict(color='red'))

# Create layout with axis labels and title
layout = go.Layout(
    title='Zuurstof en bloeddruk (Plotly)',
    xaxis=dict(title='Tijd'),
    yaxis=dict(title='Waarde'),
    showlegend=True,
    hovermode='closest'
)

# Create figure and plot
fig = go.Figure(data=[trace1, trace2], layout=layout)
fig.show()

File: Library_tests/Julia/LB_plotly.jl
using PlotlyJS
using Random

# Generate random data points
num_points = 20
x = 1:num_points
y1 = rand(5:15, num_points)  # Oxygen levels
y2 = rand(70:120, num_points)  # Blood pressure

# Create scatter traces for oxygen levels and blood pressure
trace1 = scatter(
    x = collect(x),
    y = y1,
    mode = "markers+lines",
    name = "Zuurstof",
    marker_color = "blue"
)

trace2 = scatter(
    x = collect(x),
    y = y2,
    mode = "markers+lines",
    name = "Bloeddruk",
    marker_color = "red"
)

# Create layout for the plot
layout = Layout(
    title = "Zuurstof en bloeddruk",
    xaxis_title = "Tijd",
    yaxis_title = "Waarde",
    legend = attr(x = 0, y = 1),
    hovermode = "closest"
)

# Create Plotly plot
plotly_plot = Plot([trace1, trace2], layout)

# Display the plot in the default web browser
display(plotly_plot)
